src/controller.rs

```diff
@@ -138,11 +162,11 @@ async fn reconcile(app_instance: Arc<AppInstance>, ctx: Arc<Context>) -> Result<
 
     let action = match state {
         ReconciliationState::Idle => {
-            match launch_job(&app_instance, &ctx).await {
+            match launch_job(app_instance, ctx).await {
```

Comment:
**Note to reviewers:**

These lines which relate to changing `&app_instance` to `app_instance` were all `cargo clippy` recommendations, there's quite a few as a heads up :smile: